include/RigidBody/Joints.h

```diff
@@ -432,7 +433,7 @@ class BIORBD_API Joints : public RigidBodyDynamics::Model
     /// \brief Return the total mass of the model
     /// \return The toal mass of the model
     ///
-    double mass() const; 
+    const biorbd::utils::Scalar &mass() const; 
```

Comment:
The return should not be a const ref (`biorbd::utils::Scalar mass() const;` instead of `const biorbd::utils::Scalar &mass() const;`)

Reply:
Done